templates/document.tmpl

```diff
@@ -117,7 +117,7 @@ define('conf', ['locale'], function () {
 
     conf.messages = {};
     #if $_anonymous
-    conf.messages.anonymous_user_warning = "$($_('KonText is running in anonymous mode. Please <a href=\\"%s\\" class=\\"signon\\" onclick=\\"return false;\\">sign-in</a> to unlock all the functions.') % $login_url)";
+    conf.messages.anonymous_user_warning = "$($_('KonText is running in anonymous mode. Please login or signup to unlock all the functions.'))";
     #end if
```

Comment:
I thought it was redundant as there is already a login link, and also it did not work like this. If you think it should be there, I will fix it and commit with a link.